spec/features/wiki/crud_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'rails_helper'
+
+feature 'wiki', type: :feature do
+
+  scenario "creates a wiki entry" do
+    # test data creation
+    create(:user, email: "test@email.com", password: "test-password")
+
+    # localhost:3000/users/sign_in
+    visit new_user_session_path
+    fill_in "Email", with: "test@email.com"
+    fill_in "Password", with: "test-password"
+    click_button "Log in"
```

Comment:
Lines 10-13 and repeated throughout this spec. Move them to a private method in this spec.